tests/wallet.test.js

```diff
@@ -13,21 +13,20 @@ describe('wallet:', function () {
     assertUserIsLoggedIn(browser)
   })
 
-  it('should show ETH balance', async function (done) {
+  it('should show ETH balance', function () {
     sendSomeETH(userAccount, 3.1)
-    browser.waitForClickable('.wallet-contents li:last-child .amount')
-    const amount = await browser.getText('.wallet-contents li:last-child .balance', false)
-    assert.equal(amount, '3.10 ETH')
-    done()
+    browser.waitForVisible('.wallet-contents li:last-child .balance')
+    browser.waitUntil(() => {
```

Comment:
This change (and the one below) is what I believe to be a resiliency improvement/guard against timeout

Reply:
+1 for the waitUntil